lens/fetcher.go

```diff
@@ -34,7 +37,9 @@ type lensedFetcher struct {
 
 	txn datastore.Txn
 
-	col client.Collection
+	identity immutable.Option[string]
+	acp      immutable.Option[acp.ACPModule]
```

Comment:
todo: I cannot see why these need to be persisted on the struct - can you please remove them?

Reply:
- [ ] Will do